docs/cli-reference.md

```diff
@@ -236,22 +234,22 @@ Global Flags:
   -v, --verbose            Enable verbose output
 ```
 
-### `wego app remove`
+### `gitops app remove`
 
-Remove an application from a wego cluster so it will no longer be managed via GitOps
+Remove an application from a gitops cluster so it will no longer be managed via GitOps
 
 ```console
 Usage:
-  wego app remove [--private-key <keyfile>] <app name> [flags]
+  gitops app remove [--private-key <keyfile>] <app name> [flags]
 
 Examples:
 
-  # Remove application from wego control via immediate commit
-  wego app remove podinfo
+  # Remove application from gitops control via immediate commit
+  gitops app remove podinfo
 
 
 Flags:
-      --dry-run              If set, 'wego remove' will not make any changes to the system; it will just display the actions that would have been taken
+      --dry-run              If set, 'gitops remove' will not make any changes to the system; it will just display the actions that would have been taken
```

Comment:
maybe `gitops app remove` instead of `gitops remove` ? :)

Reply:
Sure. I didn't go back and change that behavior in the weave-gitops repo when I updated the docs. I can update both of them.